docs/articles_en/get_started/installing-openvino-overview/installing-openvino-linux-header/installing-openvino-zypper.rst

```diff
@@ -0,0 +1,196 @@
+.. {#openvino_docs_install_guides_installing_openvino_zypper}
+
+Install OpenVINO™ Runtime on Linux From ZYPPER Repository
+========================================================
+
+
+.. meta::
+   :description: Learn how to install OpenVINO™ Runtime on Linux operating
+                 system, using the ZYPPER repository.
+
+.. note::
+
+   Note that the ZYPPER distribution:
+
+   * offers both C/C++ and Python APIs
```

Comment:
as far as I can see https://build.opensuse.org/projects/home:cabelo:intel/packages/openvino/repositories/15.5/binaries python package is not provided, or I'm missing something?

Reply:
It's true, but I copied the structure from Fedora, which doesn't have python apis. See the evidence below:
[root@fedora-machine /]# rpm -qa|grep openvino
libopenvino-2023.3.0-2023.3.0.13775-1.x86_64
libopenvino-auto-batch-plugin-2023.3.0-2023.3.0.13775-1.x86_64
libopenvino-auto-plugin-2023.3.0-2023.3.0.13775-1.x86_64
libopenvino-hetero-plugin-2023.3.0-2023.3.0.13775-1.x86_64
libopenvino-intel-cpu-plugin-2023.3.0-2023.3.0.13775-1.x86_64
libopenvino-ir-frontend-2023.3.0-2023.3.0.13775-1.x86_64
libopenvino-onnx-frontend-2023.3.0-2023.3.0.13775-1.x86_64
libopenvino-paddle-frontend-2023.3.0-2023.3.0.13775-1.x86_64
libopenvino-pytorch-frontend-2023.3.0-2023.3.0.13775-1.x86_64
libopenvino-tensorflow-frontend-2023.3.0-2023.3.0.13775-1.x86_64
libopenvino-tensorflow-lite-frontend-2023.3.0-2023.3.0.13775-1.x86_64
libopenvino-devel-2023.3.0-2023.3.0.13775-1.x86_64
libopenvino-intel-gpu-plugin-2023.3.0-2023.3.0.13775-1.x86_64
openvino-libraries-2023.3.0-2023.3.0.13775-1.noarch
openvino-libraries-devel-2023.3.0-2023.3.0.13775-1.noarch
openvino-samples-2023.3.0-2023.3.0.13775-1.noarch
openvino-2023.3.0-2023.3.0.13775-1.noarch



I can put it on the roadmap and now remove only mentioning API in C++, is that ok with you @ilya-lavrenov  ?